js/server/types/User.js

```diff
@@ -0,0 +1,228 @@
+const _ = require('lodash');
```

Comment:
I really like what you're doing here to consolidate everything type-wise

Reply:
Thanks 👊  It also speeds up V8 quite a bit